db/rdbms/src/main/java/io/camunda/db/rdbms/RdbmsConfiguration.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms;
+
+import io.camunda.db.rdbms.queue.ExecutionQueue;
+import io.camunda.db.rdbms.service.ExporterPositionRdbmsService;
+import io.camunda.db.rdbms.service.ProcessDefinitionRdbmsService;
+import io.camunda.db.rdbms.service.ProcessInstanceRdbmsService;
+import io.camunda.db.rdbms.service.VariableRdbmsService;
+import io.camunda.db.rdbms.sql.ExporterPositionMapper;
+import io.camunda.db.rdbms.sql.ProcessDefinitionMapper;
+import io.camunda.db.rdbms.sql.ProcessInstanceMapper;
+import io.camunda.db.rdbms.sql.VariableMapper;
+import io.camunda.zeebe.scheduler.ActorScheduler;
+import java.io.IOException;
+import java.util.Properties;
+import javax.sql.DataSource;
+import liquibase.integration.spring.MultiTenantSpringLiquibase;
+import org.apache.ibatis.mapping.VendorDatabaseIdProvider;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.type.JdbcType;
+import org.mybatis.spring.SqlSessionFactoryBean;
+import org.mybatis.spring.mapper.MapperFactoryBean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+@Configuration
```

Comment:
To stay in line with the pattern we use in terms of Spring, these Spring configuration classes must be moved to the `dist` module. Also, the Spring dependency should be removed from the `pom.xml`.